src/api/hourliesAPI.ts

```diff
@@ -0,0 +1,37 @@
+import { Station } from 'api/stationAPI'
+import axios from 'api/axios'
+
+interface HourlyReading {
+  datetime: string
+  temperature: number
+  relative_humidity: number
+  wind_speed: number
+  wind_direction: number
+  barometric_pressure: number
+  precipitation: number
+  ffmc: number
+  isi: number
+  fwi: number
```

Comment:
If these can be null values you can do 
ffmc?: number
isi?: number
fwi?: number